### ISSUE TITLE
Alt text from images should transfer to mastodon images

### ISSUE THREAD

**Issue by pogmommy** [pogmom/TwitToMast#6]
right now, cross-posted mastodon images do not post with any alt image data, reducing accessibility for users unable to view images and relying on screen readers. TwitToMast needs to copy any existing alt image data while reposting images

**Comment by pogmommy** [comment]
marking as bug despite not being implemented yet because this is not an optional feature.

**Comment by pogmommy** [comment]
added in 4fdd9f0
